tests/unit/AgmExtensionEvaluation.spec.ts

```diff
@@ -36,10 +31,222 @@ describe('ExtensionRequest', () => {
     expect(rows.at(0).find('.col-sm-9').text()).toBe('2023')
 
     expect(rows.at(1).find('.col-sm-3').text()).toBe('Duration of Extension')
-    expect(rows.at(1).find('.col-sm-9').text()).toBe('Unknown')
+    expect(rows.at(1).find('.col-sm-9').text()).toBe('')
 
     expect(rows.at(2).find('.col-sm-3').text()).toBe('Due date for this AGM')
-    expect(rows.at(2).find('.col-sm-9').text()).toBe('Unknown')
+    expect(rows.at(2).find('.col-sm-9').text()).toBe('')
+
+    wrapper.destroy()
+  })
+
+  it('is eligible', () => {
+    // init store
+
+    const wrapper = mount(AgmExtensionEvaluation, {
+      propsData: {
+        data: {
+          agmYear: 2023,
+          isEligible: true,
+          alreadyExtended: false,
+          requestExpired: false
+        }
+      },
+      vuetify
+    })
+
+    const rows = wrapper.findAll('.content .row')
+
+    expect(rows.at(0).find('.col-sm-9').text()).toBe('2023')
+
+    expect(rows.at(1).find('.col-sm-9').text()).toBe('months')
+
+    expect(rows.at(2).find('.col-sm-9').text()).toBe('')
+
+    wrapper.destroy()
+  })
+
+  it('is eligible, setting the already extended and request expired to true', () => {
+    // init store
+
+    const wrapper = mount(AgmExtensionEvaluation, {
+      propsData: {
+        data: {
+          agmYear: 2023,
+          isEligible: true,
+          alreadyExtended: true,
+          requestExpired: true
+        }
+      },
+      vuetify
+    })
+
+    const rows = wrapper.findAll('.content .row')
+
+    expect(rows.at(0).find('.col-sm-9').text()).toBe('2023')
+
+    expect(rows.at(1).find('.col-sm-9').text()).toBe('months')
+
+    expect(rows.at(2).find('.col-sm-9').text()).toBe('')
+
+    wrapper.destroy()
+  })
+
+  it('is eligible, setting the already extended to false request expired to true', () => {
+    // init store
+
+    const wrapper = mount(AgmExtensionEvaluation, {
+      propsData: {
+        data: {
+          agmYear: 2023,
+          isEligible: true,
+          alreadyExtended: false,
+          requestExpired: true
+        }
+      },
+      vuetify
+    })
+
+    const rows = wrapper.findAll('.content .row')
+
+    expect(rows.at(0).find('.col-sm-9').text()).toBe('2023')
+
+    expect(rows.at(1).find('.col-sm-9').text()).toBe('months')
+
+    expect(rows.at(2).find('.col-sm-9').text()).toBe('')
+
+    wrapper.destroy()
+  })
```

Comment:
This is essentially the same test as above, since isEligible=True gives the same results regardless of alreadyExtended.